src/distance.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {maxDistance};
+const {
+    assertArray,
+    assertArrayItemsInstanceOf,
+    assertArrayLengthEq,
+    assertArraysOfArrayNotEmpty,
+    assertArrayNotEmpty
+} = require("./typeassert");
+
+/**
+ * @summary Measures the max distance between all criterias.
+ * @param {number[][]} criteria 2d array with students and their criterias
+ * @returns {number} the calculated score
+ */
+function maxDistance(criteria){
+    assertArray(criteria);
+    assertArrayNotEmpty(criteria);
+    assertArrayItemsInstanceOf(criteria, Array);
+    assertArraysOfArrayNotEmpty(criteria);
+    assertArrayLengthEq(...criteria);
+
+    let value = 0;
+    for(let i = 0; i < criteria[0].length; i++){
```

Comment:
Her kan vi benytte Math.min(...criteria) og Math.max(...criteria) i stedet for endnu et loop

Reply:
Og et reduce kan benyttes til at fjerne det første for loop

Reply:
Vi har fjernet det ene forloop, og bruger spread operatoren til at finde min of max